src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -839,11 +839,33 @@ class ParseLiveQueryServer {
       return;
     }
 
+    const subscription = subscriptionInfo.subscription;
+    const className = subscription.className;
+    const trigger = getTrigger(className, 'beforeUnsubscribe', Parse.applicationId);
+    if (trigger) {
+      const auth = await this.getAuthFromClient(client, request.requestId, request.sessionToken);
```

Comment:
In the other uses, `getAuthFromClient` is wrapped in the try / catch block. I don't think it throws, but could you add it to the try / catch, or add a test case for `beforeUnsubscribe` without the `user.signUp`?